mam/api/api.c

```diff
@@ -761,30 +761,45 @@ retcode_t mam_api_deserialize(trit_t const *const buffer, size_t const buffer_si
   return RC_OK;
 }
 
-retcode_t mam_api_save(mam_api_t const *const api, char const *const filename) {
+retcode_t mam_api_save(mam_api_t const *const api, char const *const filename, trits_t encryption_key) {
```

Comment:
What if we use the API seed as encryption key ?

Reply:
As we discussed of line,
user can use his/her seed, but using the additional arg, that way he/she can choose which ever key to encrypt, also, with the arg they can choose not to encrypt by providing `trits_null()`

Reply:
Yes, choosing if we want to encrypt of not is actually pretty cool!